src/puptoo/process/__init__.py

```diff
@@ -63,8 +60,10 @@ def extract(msg, extra, remove=True):
         except Exception as e:
             logger.exception("Failed to extract facts: %s", str(e), extra=extra)
             facts["error"] = str(e)
-        finally:
-            facts = postprocess(facts)
-            metrics.msg_processed.inc()
```

Comment:
These metrics are going to be incremented in the case of a success or failure due to the finally.